arch/arm/src/stm32wl5/stm32wl5_exti_gpio.c

```diff
@@ -121,9 +253,60 @@ static int stm32wl5_exti0_15_isr(int irq, void *context, FAR void *arg)
 int stm32wl5_gpiosetevent(uint32_t pinset, bool risingedge, bool fallingedge,
                          bool event, xcpt_t func, void *arg)
 {
+  struct gpio_callback_s *shared_cbs;
   uint32_t pin = pinset & GPIO_PIN_MASK;
-  uint32_t exti = 1 << pin;
-  int      irq = STM32WL5_IRQ_EXTI0 + pin;
+  uint32_t exti = (1 << pin);
```

Comment:
Minor

```suggestion
  uint32_t exti = 1 << pin;
```

Reply:
Ahh, right, I've changed the another "1 << pin" to "(1 << pin)" instead the other way around xD

Reply:
Anyway that was minor, so I approved anyway

Reply:
Minor or no minor, if we leave these minor stylish things in code, it will look inconsistent like it came from a big corporation:)